src/data_utils/other.ts

```diff
@@ -0,0 +1,85 @@
+/**
```

Comment:
This file should be renamed to `StringValidation.ts` (or something more fitting, since it also checks for arrays, maybe just `Validation.ts`?) and should be placed under `src/utils/`.

Reply:
would you use that module somewhere on the router part? or, anywhere else outside the `data_utils` folder really

Reply:
Doesn't matter. Files should be organized according to their functionality, not their use case.

Reply:
implemented